### DISCUSSION TITLE
[BUG] Sync problem

### DISCUSSION THREAD

**Discussion by potato1992** [Chia-Network/chia-blockchain#5314] [Support]
I fully understand that there are so many reasons that can cause the **not sync** situation. 

But, when I try to connect to one peers, even another peer owned by myself, nothing happens at all. Can you just put some log on the GUI to show what the hell have happened?????? 

Do you really have product manager? Please be serious, this is not school homework.

**Comment by Chiqui1234ok** [comment]
My Chia GUI says "Not synced" and the log says:

```bash
2021-05-16T20:10:14.596 full_node full_node_server        : ERROR    Exception:  <class 'concurrent.futures._base.CancelledError'>, closing connection None. Traceback (most recent call last):
  File "chia\server\server.py", line 531, in api_call
  File "asyncio\tasks.py", line 435, in wait_for
concurrent.futures._base.CancelledError
```

Python error?

Funny thing: I delete *db* and *log* folder because it get stuck. Then re-launch Chia and above error happened :(

**Comment by paritad99** [comment]
i same you guy, i install python 3.7 not error

**Comment by sky168888** [comment]
i got same problem, i delete .chia folder and re-launch again. start syncing again and see what happen after fully sync.

**Comment by Chiqui1234ok** [comment]
Ok, I removed all inside .chia/mainnet, except wallet's folder. The GUI is slow and weird, so I'm moving back to CLI.

```bash
chia show -a chiadb.net:8444
```

Above command add *chiadb.net* as a peer. After 5 or 10 minutes it starts syncing :)
Then, you can check the blockchain sync with

```bash
chia show -s
```

PD: Sync your clock and GMT location 👍 Otherwise it will fail somehow

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
